### ISSUE TITLE
Improve the UI

### ISSUE THREAD

**Issue by andymachs** [amir1376/ab-download-manager#289]
Improve the UI.

![image](https://github.com/user-attachments/assets/e20329f9-2a2d-4411-af77-29d1d8a283b2)

**Comment by andymachs** [comment]
Please consider improve UI.

**Comment by amir1376** [comment]
Thank you.

**Comment by andymachs** [comment]
@amir1376,  Yes, if adjust and improve UI by details further, I think it looks like more modern and beautiful.

**Comment by andymachs** [comment]
The UI of search window is not good, poor visual effect. 
Also it doesn't look good where it's placed. Could you improve it further?

![image](https://github.com/user-attachments/assets/40d92645-001d-4b31-b21e-ae8f0eb199cb)
